cloud/passportauth/src/user/UserService.ts

```diff
@@ -0,0 +1,32 @@
+
+/**
+ * User service interface that defines a set of requirements (methods) for mapping particular fields from the
+ * user object returned from the repository.
+ *
+ * @see UserRepository
+ */
+export interface UserService {
+  /**
+   * Retrieves unique login id
```

Comment:
As per Clean Code ([really good book, recommending that one)](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882) this comments are just noise as their duplicating method names and give no additional value to users.